android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -232,6 +232,16 @@ fun setLogHandler(
     Purchases.logHandler = logHandler
 }
 
+/**
+ * Sets a log handler and forwards all logs to completion function.
+ *
+ * @param callback Gets a map with two keys, a `logLevel` which  is one of the ``LogLevel`` name lowercased,
+ * and a `message`, with the log message.
+ */
+fun setLogHandler(callback: (logDetails: Map<String, String>) -> Unit) {
```

Comment:
Hmm should we deprecate/remove the old `setLogHandler` method? I believe it only exists for Android.

Reply:
I will remove it, I don't think it's used